integrations/apk_analyzers/compose-tests.yml

```diff
@@ -9,13 +9,4 @@ services:
     build:
       context: ../integrations/apk_analyzers
       dockerfile: Dockerfile
-    container_name: intelowl_apk_analyzers
-    restart: unless-stopped
-    expose:
-      - "4004"
-    env_file:
-      - env_file_integrations
-    volumes:
-      - generic_logs:/var/log/intel_owl
-    depends_on:
-      - uwsgi
```

Comment:
why delete all these attributes from the configuration ?

Reply:
I removed duplicated code between compose files and adjusted start.py script to properly leverage override files.